pkgs/development/python-modules/aws-sam-translator/default.nix

```diff
@@ -13,7 +13,7 @@
 
 buildPythonPackage rec {
   pname = "aws-sam-translator";
-  version = "1.42.0";
+  version = "1.45.0";
```

Comment:
The translator bump should be a separate commit

please look at https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes